windows/wsl_distro_install_uninstall/install_wsl.yml

```diff
@@ -132,3 +132,10 @@
       - wsl_default_version | int == 1
     fail_msg: "WSL default version is: {{ wsl_default_version }}, not the expected version: 1"
   when: skip_set_wsl_ver
+
+- name: "Check if the WSL distribution is installed"
+  include_tasks: check_wsl_distro.yml
+
+- name: "Install the WSL distribution"
+  include_tasks: install_wsl_distro.yml
+  when: not wsl_distro_installed
```

Comment:
The value of `wsl_distro_installed` could be changed in task files included by `install_wsl_distro.yml`. It might affect the task following `wsl_distro_installed` setting.

Reply:
That's as expected. 

The value of wsl_distro_installed will be set to true once.  If the distribution is installed along with WSL, it'll be set to true in install_wsl.yml, otherwise the distribution will be installed in install_wsl_distro.yml and the value will be set to true here.